### DISCUSSION TITLE
兄弟们，你们的简现在什么练度了

### DISCUSSION THREAD

**Discussion by Loxic123** [share121/inter-knot#5272] [绝区零]
我已经突破63万了
![image](https://github.com/user-attachments/assets/32eba035-f39e-4951-93a6-a11959bbf516)

**Comment by cuijiaqi0406** [comment]
刀耕火焚112w，怎么样

**Comment by SakuraLainMaid** [comment]
![Screenshot_20240921_175444](https://github.com/user-attachments/assets/3f2e6d1a-edcc-4e59-9d34-c2d5f7109ce9)
谢邀，1135w

**Comment by yansai0v0** [comment]
单人才二十w出头😭
